pluton/cluster.go

```diff
@@ -43,6 +47,7 @@ type Info struct {
 	KubeletTag      string // e.g. v1.5.3_coreos.0
 	Version         string // e.g. v1.5.3+coreos.0
 	UpstreamVersion string // e.g. v1.5.3
+	Options                // pipe through how test was configured
```

Comment:
Personally I think `Options: pluton.Options{` when the defaults aren't applicable seems more awkward than not being able to use defaults.

You can leave it as-is for now to see if that sort of defaults usage proves valuable or not but unless Options grows in complexity I suspect you may be best served just forcing InitialMasters/Workers to `1` in the setup code if they are left as zero since largely that's the value most tests are using anyway.